sdcm/utils/common.py

```diff
@@ -2247,3 +2248,21 @@ def walk_thru_data(data, path: str) -> Any:
             continue
         current = current.get(name, None)
     return current
+
+
+def update_authenticator(nodes, authenticator='AllowAllAuthenticator', restart=True):
+    """
+    Update the authenticator of nodes, restart the nodes to make the change effective
+    """
+    for node in nodes:
+        node.remoter.sudo(
+            f"sed -ie 's/^authenticator:.*/authenticator: {authenticator}/g' /etc/scylla/scylla.yaml")
+        if restart:
+            if authenticator == 'com.scylladb.auth.SaslauthdAuthenticator':
+                node.run_cqlsh(f'ALTER ROLE \'{LDAP_USERS[0]}\' with password=\'{LDAP_PASSWORD}\'')
+            node.use_saslauthd_authenticator = authenticator == 'com.scylladb.auth.SaslauthdAuthenticator'
+            node.parent_cluster.params['are_ldap_users_on_scylla'] = node.use_saslauthd_authenticator
+            node.restart_scylla_server()
+            node.wait_db_up()
+            # Wait 2 mins to restart another node
+            time.sleep(120)
```

Comment:
why do we need to sleep 2 minutes here? who said 2 minutes will be enough?
it could either delay us unnecessarily or be not enough... 
if you need to wait for all nodes to be restarted to continue, then it should be called from inside a function that will move forward only when all nodes have been updated

Reply:
This can be removed, it's used to avoid c-s fail in rolling restart all nodes one by one.
Actually the c-s fail was caused by other reason which had been fixed.

I will remove the sleep.